src/gui/LayoutMapper.h

```diff
@@ -21,23 +21,20 @@
 
 #include "XournalType.h"
 
-namespace std {
-template <>
-class hash<std::pair<size_t, size_t>> {
-public:
-    size_t operator()(std::pair<size_t, size_t> const& data) const {
-        size_t seed = 0;
-        boost_c::hash_combine(seed, data.first);
-        boost_c::hash_combine(seed, data.second);
```

Comment:
Since these were the only two occurences of `hash_combine` it would make sense to remove `hashcombine.h` and the license reference in `copyright.txt` (lines 89-93)

Reply:
is it ok to just remove the entry or do I have to execute the python script?

Reply:
I think it is OK. Running the python script would only have told you that the `copyright.txt` should be updated. We can update the `latest_commit_hash` another time.